files/system/usr/share/ublue-os/just/70-secureblue.just

```diff
@@ -534,28 +534,34 @@ audit-secureblue:
         for f in ${!flatpaks[@]}; do
             has_network=false
             has_x11=false
+            has_hardened_malloc=true
+            status=$STATUS_SUCCESS
             fullref=${flatpaks["$f"]}
             permissions=$(flatpak info --show-permissions "$fullref")
             if hasPermission "$permissions" "shared" "network"; then
                 has_network=true
+                [[ ! "$status" != "$STATUS_FAILURE" ]] && status="$STATUS_WARNING"
             fi
-            if hasPermission "$permissions" "sockets" "x11" && ! hasPermission "$permissions" "sockets" "fallback-x11"  ]]; then
+            if hasPermission "$permissions" "sockets" "x11" && ! hasPermission "$permissions" "sockets" "fallback-x11"; then
                 has_x11=true
+                status="$STATUS_FAILURE"
             fi
-            flatpak_test_string="Auditing $f"
-            if [[ ! $has_network == "true" && ! $has_x11 == "true" ]]; then
```

Comment:
why was this removed?

Reply:
Part of the change to how the status is managed. Rather than checking something like 2^n states (less in reality, but still), each permission check can only move the status in one direction: SUCCESS > WARNING or FAILURE, but never FAILURE > WARNING or SUCCESS.

Reply:
interesting, thanks for clarifying